product_docs/docs/pgd/5/node_management/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Node management
+navTitle: Node management
+navigation:
+ - creating_and_joining
+ - witness_nodes
+ - logical_standby_nodes
+ - physical_standby_nodes
+ - groups_and_subgroups
+ - subscriber_only_groups
+ - viewing_topology
+ - removing_nodes_and_groups
+ - heterogeneous_clusters
+ - connections_dsns_and_ssl
+ - decoding_worker
+ - replication_slots
+ - node_recovery
```

Comment:
What are the criteria for sorting these topics in how they are ordered now? Would reviewing the grouping and arrangement make sense based on how often these tasks occur? Or is that planned later, after this initial structure has been established?